package.json

```diff
@@ -70,5 +70,9 @@
     "husky": "^6.0.0",
     "prettier": "^2.2.1",
     "pretty-quick": "^3.1.0"
+  },
+  "resolutions": {
```

Comment:
What are these for?

Reply:
I got the eslint plugin in my vscode and it would just show me errors in some files in line0: Parsing error: Cannot read property 'map' of undefined

So I googled a bit and found an explanation:
>This error occurs because react-scripts has a direct dependency on the 2.xx range  of @typescript-eslint/parser and @typescript-eslint/eslint-plugin.

https://stackoverflow.com/a/63919395

I think alternatively we could try to update react-scripts. I'll look into it or write a comment on why I put it there.

Reply:
Yes! I was wondering how to get rid of these!

Reply:
And I also got the same errors when trying to make eslint a dependency. Seems they don't work together somehow. I was hoping to use eslint as the `yarn lint` command for our github actions too.